homeassistant/components/dwd_weather_warnings/sensor.py

```diff
@@ -184,7 +185,10 @@ def __init__(self, region_name):
             "jsonp=loadWarnings",
         )
 
-        self._rest = RestData("GET", resource, None, None, None, True)
+        # a dummy User-Agent is necessary to trick the rest api endpoint (see #29496)
```

Comment:
There is no dummy stuff anymore. Also I'd not consider this tricking anybody. It's just a requirement of the sensor for a request to have a user agent.

Reply:
Corrected it with https://github.com/home-assistant/home-assistant/pull/29596/commits/abab11719fad0815252a5d585edbc68a245c17a4